.azure_pipelines_snp.yml

```diff
@@ -33,28 +33,47 @@ jobs:
       vmImage: ubuntu-20.04
     steps:
       - script: |
+          echo -e "\nGenerating an SSH key for use with the executors..."
```

Comment:
```suggestion
          set -ex
          echo -e "\nGenerating an SSH key for use with the executors..."
```

Reply:
Sounds good, can I do a proper pass in https://github.com/microsoft/CCF/pull/4662?